# ControlType, the type of target that needs to be detected
# 0: car
# 1: medicalBag
# 2: colorPerson
# 3: redPerson
# 4: blackCircle
int8 ControlType

# start/stop
bool Start
---------------
bool isOk
